packages/wrapped-keys/src/lib/utils.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  AccessControlConditions,
+  AuthSig,
+  SessionKeySignedMessage,
+  SessionSigsMap,
+} from '@lit-protocol/types';
+import { log } from '@lit-protocol/misc';
+import { CHAIN_ETHEREUM } from './constants';
+import { ethers } from 'ethers';
+// import { log } from 'console';
+
+export function getFirstSessionSig(pkpSessionSigs: SessionSigsMap): AuthSig {
+  const keys = Object.keys(pkpSessionSigs);
+  if (keys.length === 0) {
+    throw new Error(
+      `Invalid pkpSessionSigs, length zero: ${JSON.stringify(pkpSessionSigs)}`
+    );
+  }
+
+  const firstSessionSig: AuthSig = pkpSessionSigs[keys[0]];
```

Comment:
Do we check in the lambdas that the node which signed the first signature is still in the active signing set? I feel there might be issues with old sessions which are signed from nodes which have been kicked will get validated in the Lamda context.

Reply:
As long as your sessionSig is valid what's the issue?

Reply:
If the node is not in the active set. In theory we should not trust the signature from that node. But since the sdk combines T number of shares where T is the network threshold for each `auth sig` composing the `session signature` map it only matters if we want to make sure an active network participant's mapped signatrue is being sent to the lamda. If we want to enforce this we can check the current `connectedUrls` in the sdk and see if the url from the authSig chosen from the map is indeed active.

Reply:
The inner authSig is a collection from all the nodes so even if a node is kicked out the innerAuthSig is valid and we validate that in the lambda. We check that the inner AuthSig is signed by the BLS root key?